src/components/Thanks/ThanksPageSingleVersion.vue

```diff
@@ -145,11 +158,34 @@ const scrollToReceipt = () => {
 	setTimeout(() => {
 		receiptSection.value?.orderConfirmationContainer?.scrollIntoView({ behavior: 'smooth' });
 	}, 500);
+
+	$kvTrackEvent(
```

Comment:
The specific event here is clicking view pdfs. I'm wondering if we should just fire that event from `KivaCards.vue` to have it closer to the source of the event.

Reply:
Just same action for both cases thought, but fired from different component...

Reply:
It's a slightly different action, so we'll want to have it specific to the view PDFs button. This method is only used by that component currently.